Engineering Village.js

```diff
@@ -8,51 +8,51 @@
 	"priority": 100,
 	"inRepository": true,
 	"translatorType": 4,
-	"browserSupport": "gcsb",
-	"lastUpdated": "2017-02-12 17:40:59"
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-10-07 10:02:04"
 }
 
 function detectWeb(doc, url) {
-	Z.monitorDOMChanges(doc.getElementById("resultsarea"), {childList: true});
-	var downloadLink = doc.getElementById('oneclickDL');
-	if (downloadLink && getDocIDs(downloadLink.href)) {
+	Z.monitorDOMChanges(doc.getElementById("ev-application"), {childList: true});
+	var printlink = doc.getElementById('printlink');
+	if (url.includes("/search/doc/") && printlink && getDocIDs(printlink.href)) {
 		return "journalArticle";
 	}
-	
-	if (getSearchResults(doc, true)) {
+	if ((url.includes("quick.url?") || url.includes("expert.url?") || url.includes("thesaurus.url?")) && getSearchResults(doc, true)) {
 		return "multiple";
 	}
+
+
 }
 
+
 function getDocIDs(url) {
 	var m = url.match(/\bdocidlist=([^&#]+)/);
 	if (!m) return false;
 	
 	return decodeURIComponent(m[1]).split(',');
 }
 
+
 function getSearchResults(doc, checkOnly) {
-	var rows = doc.getElementsByClassName('result'),
+	var rows = doc.querySelectorAll('div[class*=result-row]'),
 		items = {},
 		found = false;
-	
 	for (var i=0; i<rows.length; i++) {
 		var checkbox = rows[i].querySelector('input[name="cbresult"]');
 		if (!checkbox) continue;
-		
 		var docid = checkbox.getAttribute('docid');
 		if (!docid) continue;
-		
-		var title = rows[i].querySelector('h3.resulttitle');
+		var title = rows[i].querySelector('h3.result-title');
 		if (!title) continue;
 		
 		if (checkOnly) return true;
 		found = true;
 		
 		items[docid] = {
 			title: ZU.trimInternal(title.textContent),
-			checked: checkbox.checked
-		}
+			checked: checkbox.checked // TODO is this still needed?
```

Comment:
It looks for me that this is  not anymore needed and therefore we can also switch to our default template for the function `getSearchResults`. Can you try that out @adam3smith ?